src/components/app.js

```diff
@@ -22,7 +22,7 @@ class App extends React.Component {
       console.log('[Vega-Editor] Received Message', evt.origin, data);
       // send acknowledgement
       var parsed = JSON.parse(data.spec);
-      data.spec = JSON.stringify(parsed, null, 2);
+      data.spec = JSON.stringify(parsed);
```

Comment:
Since the spec is formatted in `updateVegaSpec` or `updateVegaLiteSpec`, it doesn't matter if we format it here or not. 
I'm not sure why the code here parses the spec then stringifys it, I think we should just remove
```javascript
var parsed = JSON.parse(data.spec);
data.spec = JSON.stringify(parsed);
```